github_actions/deploy_cluster/action.yml

```diff
@@ -89,7 +89,7 @@ inputs:
   image_id:
     description: 'Image ID used for all VMs'
     required: false
-    default: "ami-9b41b457"
+    default: "ami-1111f1b5"
```

Comment:
Are you sure about this ID ?

Reply:
Yes 
https://docs.outscale.com/fr/userguide/R%C3%A9f%C3%A9rence-des-OMI-officielles.html#_us-east-2
the [Ubuntu-22.04-2023.12.04-0](https://docs.outscale.com/fr/userguide/Ubuntu-22.04-2023.12.04-0.html) in us-east-2